site/src/routes/about/+page.svelte

```diff
@@ -54,10 +54,18 @@
         Jupiterp is managed by our Core Team:
     </p>
     <ul class={pClasses + ' list-disc ml-8'}>
-        <li>Andrew Cupps 
+        <li>
+            <a href='https://atcupps.github.io' target='_blank'
+                                            class='text-orange underline'>
+                AT Cupps 
```

Comment:
why is yours a different name format than mine

Reply:
It's just because your link takes up more space so I started a new line, but I can make them the same in my next commit

Reply:
Done